tensorflow_probability/python/bijectors/rational_quadratic_spline.py

```diff
@@ -85,26 +90,29 @@ def __call__(self, x, nunits):
         def _bin_positions(x):
           out_shape = tf.concat((tf.shape(x)[:-1], (nunits, self._nbins)), 0)
           x = tf.reshape(x, out_shape)
-          return tf.math.softmax(x, axis=-1) * (2 - self._nbins * 1e-2) + 1e-2
+          return tf.math.softmax(x, axis=-1) * (
+                self._interval_width - self._nbins * self._min_bin_width) + \
+                 self._min_bin_width
```

Comment:
Nit: Google's style guide prefers implicit line joining inside parentheses instead of backslash line continuation: https://google.github.io/styleguide/pyguide.html#32-line-length (i.e. the last line of this block could be `) + self._min_bin_width`